package.json

```diff
@@ -122,8 +122,11 @@
     "prepack": "npm run clean && npm run compile && oclif-dev manifest && oclif-dev readme",
     "pretest": "npm run depcheck && npm run lint",
     "lint": "eslint \"**/*.ts\" --quiet --fix",
-    "test:debug": "TS_NODE_COMPILER_OPTIONS='{\"rootDir\":\".\"}' nyc mocha \"test/**/*.test.ts\"",
-    "test": "TS_NODE_COMPILER_OPTIONS='{\"rootDir\":\".\"}' nyc mocha \"test/**/*.test.ts\" --reporter=xunit --reporter-options output=reports/generator.xml",
+    "testrunner": "TS_NODE_COMPILER_OPTIONS='{\"rootDir\":\".\"}' mocha",
```

Comment:
Is there any way to configure a tsconfig for the tests directory so that this isn't necessary?

Reply:
fixed